lib/managers/files.js

```diff
@@ -575,6 +575,73 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+
+ /**
+  * Enum of valid actions for locking and unlocking files
+  *
+  * @enum {LockType}
+  */
+Files.prototype.lockType = {
```

Comment:
I think in this case, since the user won't be passing the constants anywhere, these should probably just be local constants in this file, not exposed on the prototype.